logging/unit_tests/test_client.py

```diff
@@ -581,21 +582,32 @@ def test_get_default_handler_container_engine(self):
 
         self.assertIsInstance(handler, ContainerEngineHandler)
 
-    def test_get_default_handler_general(self):
+    @mock.patch('copy.deepcopy')
+    def test_get_default_handler_general(self, deepcopy):
         from google.cloud.logging.handlers import CloudLoggingHandler
 
+        http_mock = mock.Mock()
+        deepcopy.return_value = http_mock
+
+        credentials = _Credentials()
         client = self._make_one(project=self.PROJECT,
-                                credentials=_Credentials(),
+                                credentials=credentials,
                                 use_gax=False)
         handler = client.get_default_handler()
         self.assertIsInstance(handler, CloudLoggingHandler)
+        self.assertTrue(credentials.authorized, http_mock)
 
-    def test_setup_logging(self):
+    @mock.patch('copy.deepcopy')
+    def test_setup_logging(self, deepcopy):
```

Comment:
Since you're using `mock` already, ditch `_Monkey`? (You don't have to if it's too big a pain.)